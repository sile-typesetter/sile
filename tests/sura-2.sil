\begin[papersize=a6,direction=RTL]{document}
\nofolios
\font[family=Amiri Quran,size=20pt]
\use[module=packages.linespacing]
\set[parameter=linespacing.method,value=fixed]
\set[parameter=linespacing.fixed.baselinedistance,value=75pt]
 الۤمۤ ۝١
 ذَ ٰلِكَ ٱلۡكِتَـٰبُ لَا رَیۡبَۛ فِیهِۛ هُدࣰى لِّلۡمُتَّقِینَ ۝٢
 ٱلَّذِینَ یُؤۡمِنُونَ بِٱلۡغَیۡبِ وَیُقِیمُونَ ٱلصَّلَوٰةَ وَمِمَّا رَزَقۡنَـٰهُمۡ یُنفِقُونَ ۝٣
 وَٱلَّذِینَ یُؤۡمِنُونَ بِمَاۤ أُنزِلَ إِلَیۡكَ وَمَاۤ أُنزِلَ مِن قَبۡلِكَ وَبِٱلۡءَاخِرَةِ هُمۡ یُوقِنُونَ ۝٤
 أُو۟لَـٰۤئِكَ عَلَىٰ هُدࣰى مِّن رَّبِّهِمۡۖ وَأُو۟لَـٰۤئِكَ هُمُ ٱلۡمُفۡلِحُونَ ۝٥
 إِنَّ ٱلَّذِینَ كَفَرُوا۟ سَوَاۤءٌ عَلَیۡهِمۡ ءَأَنذَرۡتَهُمۡ أَمۡ لَمۡ تُنذِرۡهُمۡ لَا یُؤۡمِنُونَ ۝٦
 \end{document}
