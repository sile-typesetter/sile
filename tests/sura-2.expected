Set paper size 	297.6377985	419.5275636
Begin page
Mx 	243.9959
My 	51.4164
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	9.6600
Mx 	4.7600
Mx 	4.3400
T	121 x=3.9000 y=-0.1800 1294 a=9.6600 121 x=1.9600 y=0.5000 1121 a=4.7600 43 a=4.3400	(الۤمۤ)
Mx 	238.1559
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1263 a=5.8400 x=-15.7200	(١)
Mx 	212.5759
T	114 w=25.5800	(۝)
Mx 	193.4531
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	9.0000
T	77 x=-1.2800 y=-0.2600 52 a=9.0000	(ذَ)
Mx 	189.2931
T	1257 w=4.1600	( ٰ)
Mx 	169.2331
Mx 	16.9000
Mx 	3.1600
T	77 x=1.6600 y=-0.2600 1181 a=16.9000 79 x=-3.2200 1178 a=3.1600	(لِكَ)
Mx 	107.4303
Mx 	17.6600
Mx 	5.8400
Mx 	4.8800
Mx 	11.7600
Mx 	3.7000
Mx 	3.5000
Mx 	4.3400
T	78 x=3.7200 y=-0.3800 472 a=17.6600 1258 a=5.8400 77 x=-3.7200 y=-0.2600 491 a=4.8800 79 x=0.7200 523 a=11.7600 118 x=0.8400 y=-0.2800 1295 a=3.7000 525 a=3.5000 102 a=4.3400	(ٱلۡكِتَـٰبُ)
Mx 	84.4675
Mx 	6.8000
Mx 	6.0400
T	679 a=6.8000 77 x=-1.4800 y=-0.2600 677 a=6.0400	(لَا)
Mx 	43.3647
Mx 	15.4400
Mx 	5.9000
Mx 	9.6400
T	112 77 x=3.0800 y=-0.2600 829 a=15.4400 118 x=-1.2000 y=-0.2800 825 a=5.9000 77 x=0.1800 y=-0.2600 53 a=7.9800 x=1.6600	(رَیۡبَۛ)
Mx 	14.8819
Mx 	7.5800
Mx 	4.8800
Mx 	5.9000
T	112 79 x=-1.4200 517 a=7.5800 495 a=4.8800 79 x=-2.3000 502 a=5.9000	(فِیهِۛ)
Mx 	249.0159
My 	126.4164
Mx 	15.2800
Mx 	9.4200
Mx 	9.0400
T	72 a=15.2800 141 x=-0.3400 y=-0.4000 497 a=9.4200 78 x=-2.5800 y=-0.3800 519 a=9.0400	(هُدࣰى)
Mx 	197.3824
Mx 	10.8400
Mx 	4.3600
Mx 	7.5600
Mx 	4.8800
Mx 	4.7400
Mx 	4.8600
Mx 	3.1600
T	77 x=-0.2400 y=-0.2600 878 a=10.8400 877 a=4.3600 79 x=-2.0000 505 a=7.5600 369 x=1.5400 y=0.2200 80 x=-2.5000 y=-0.4600 491 a=4.8800 78 x=-3.4400 y=-0.3800 802 a=4.7200 118 x=-1.5600 y=-0.2800 1188 a=5.7000 x=-0.8400 79 x=-3.2200 80 x=-1.9600 y=-0.4600 1178 a=3.1600	(لِّلۡمُتَّقِینَ)
Mx 	191.5424
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1264 a=5.8400 x=-15.7200	(٢)
Mx 	165.9624
T	114 w=25.5800	(۝)
Mx 	123.6689
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	11.2400
Mx 	2.9000
Mx 	9.7600
Mx 	2.8200
Mx 	4.3400
T	77 x=1.0600 y=-0.2600 726 a=11.2400 725 a=2.9000 79 x=-0.9000 1071 a=9.7600 369 x=-0.8600 y=0.2200 80 x=-4.9000 y=-0.4600 1067 a=2.8200 102 a=4.3400	(ٱلَّذِینَ)
Mx 	70.7354
Mx 	11.4400
Mx 	6.9000
Mx 	4.8800
Mx 	7.7800
Mx 	6.9000
Mx 	3.8000
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-3.8400 y=-0.3800 490 a=4.8800 79 x=-1.3200 528 a=7.7800 118 x=-0.7200 y=-0.2800 554 a=6.9000 78 x=-4.3200 y=-0.3800 484 a=3.8000	(یُؤۡمِنُونَ)
Mx 	14.8819
Mx 	17.6600
Mx 	4.8800
Mx 	7.3200
Mx 	3.5000
Mx 	5.5600
Mx 	5.7000
T	79 x=4.9400 472 a=17.6600 118 x=-1.9800 y=-0.2800 495 a=4.8800 77 x=-1.9600 y=-0.2600 467 a=7.3200 118 x=-2.8600 y=-0.2800 525 a=3.5000 462 a=5.5600 79 x=-2.2400 1012 a=5.7000	(بِٱلۡغَیۡبِ)
Mx 	228.4759
My 	201.4164
Mx 	11.4400
Mx 	6.9000
Mx 	9.2600
Mx 	4.8800
Mx 	7.5600
Mx 	3.8000
Mx 	10.4400
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-1.6000 y=-0.3800 529 a=9.2600 495 a=4.8800 79 x=-2.0000 505 a=7.5600 78 x=-4.3200 y=-0.3800 484 a=3.8000 77 x=1.2600 y=-0.2600 71 a=8.0000 x=2.4400	(وَیُقِیمُونَ)
Mx 	180.8874
Mx 	7.3800
Mx 	6.4000
Mx 	5.0600
Mx 	14.4400
Mx 	3.5000
Mx 	4.3400
T	77 x=-1.7800 y=-0.2600 45 a=7.3800 101 x=0.0400 y=-9.6800 577 a=6.4000 77 x=-4.1200 y=-0.2600 575 a=5.0600 369 x=7.1000 y=0.2200 80 x=3.0600 y=-0.4600 541 a=14.4400 525 a=3.5000 102 a=4.3400	(ٱلصَّلَوٰةَ)
Mx 	152.7789
Mx 	4.5800
Mx 	3.4600
Mx 	5.6000
Mx 	8.0000
T	459 a=4.5800 369 x=1.8400 y=0.2200 80 x=-2.2000 y=-0.4600 1091 a=3.4600 79 x=-1.5600 1090 a=5.6000 77 x=-1.1800 y=-0.2600 71 a=8.0000	(وَمِمَّا)
Mx 	95.4904
Mx 	10.4400
Mx 	7.8000
Mx 	5.8400
Mx 	4.8800
Mx 	5.9000
Mx 	7.9800
Mx 	7.9800
T	118 x=0.9800 y=-0.2800 527 a=10.4400 78 x=-1.6000 y=-0.3800 520 a=7.8000 1258 a=5.8400 77 x=-3.7200 y=-0.2600 490 a=4.8800 118 x=-2.0800 y=-0.2800 501 a=5.9000 77 x=-1.4800 y=-0.2600 54 a=7.9800 77 x=-1.4800 y=-0.2600 53 a=7.9800	(رَزَقۡنَـٰهُمۡ)
Mx 	46.3019
Mx 	11.4400
Mx 	6.9000
Mx 	7.5600
Mx 	7.5600
Mx 	3.3600
Mx 	5.9000
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-1.1000 y=-0.3800 505 a=7.5600 79 x=-2.0000 506 a=7.5600 688 a=3.3600 78 x=-3.0600 y=-0.3800 825 a=5.9000	(یُنفِقُونَ)
Mx 	40.4619
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1265 a=5.8400 x=-15.7200	(٣)
Mx 	14.8819
T	114 w=25.5800	(۝)
Mx 	243.6959
My 	276.4164
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	11.2400
Mx 	2.9000
Mx 	9.7600
Mx 	2.8200
Mx 	4.3400
Mx 	8.0000
T	77 x=1.0600 y=-0.2600 726 a=11.2400 725 a=2.9000 79 x=-0.9000 1071 a=9.7600 369 x=-0.8600 y=0.2200 80 x=-4.9000 y=-0.4600 1067 a=2.8200 102 a=4.3400 77 x=-1.1800 y=-0.2600 71 a=8.0000	(وَٱلَّذِینَ)
Mx 	196.5368
Mx 	11.4400
Mx 	6.9000
Mx 	4.8800
Mx 	7.7800
Mx 	6.9000
Mx 	3.8000
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-3.8400 y=-0.3800 490 a=4.8800 79 x=-1.3200 528 a=7.7800 118 x=-0.7200 y=-0.2800 554 a=6.9000 78 x=-4.3200 y=-0.3800 484 a=3.8000	(یُؤۡمِنُونَ)
Mx 	172.3776
Mx 	9.9400
Mx 	3.4600
Mx 	5.3000
T	121 x=3.9000 y=-0.1800 1304 a=7.5000 x=2.4400 77 x=-3.9200 y=-0.2600 777 a=3.4600 79 x=-2.3600 769 a=5.3000	(بِمَاۤ)
Mx 	139.2985
Mx 	12.0600
Mx 	5.9800
Mx 	4.9000
Mx 	4.6800
T	77 x=0.4800 y=-0.2600 67 a=12.0600 79 x=-1.3200 573 a=5.9800 567 a=4.9000 371 x=1.8600 y=2.7400 39 a=4.6800	(أُنزِلَ)
Mx 	104.6793
Mx 	16.4400
Mx 	4.8800
Mx 	3.5000
Mx 	4.3400
T	77 x=1.7400 y=-0.2600 521 a=16.4400 118 x=-1.9800 y=-0.2800 495 a=4.8800 77 x=-4.6000 y=-0.2600 525 a=3.5000 373 x=3.2200 y=-1.4200 41 a=4.3400	(إِلَیۡكَ)
Mx 	73.5002
Mx 	9.9400
Mx 	7.7800
Mx 	8.0000
T	121 x=3.9000 y=-0.1800 1304 a=7.5000 x=2.4400 77 x=-1.4800 y=-0.2600 528 a=7.7800 77 x=-1.1800 y=-0.2600 71 a=8.0000	(وَمَاۤ)
Mx 	40.4210
Mx 	12.0600
Mx 	5.9800
Mx 	4.9000
Mx 	4.6800
T	77 x=0.4800 y=-0.2600 67 a=12.0600 79 x=-1.3200 573 a=5.9800 567 a=4.9000 371 x=1.8600 y=2.7400 39 a=4.6800	(أُنزِلَ)
Mx 	14.8819
Mx 	12.3000
Mx 	7.7800
T	530 a=12.3000 79 x=-1.3200 528 a=7.7800	(مِن)
Mx 	251.7359
My 	351.4164
Mx 	16.2600
Mx 	3.9800
Mx 	4.8800
Mx 	5.9000
T	77 x=1.9400 y=-0.2600 1186 a=16.2600 79 x=-5.2200 1183 a=3.9800 118 x=-1.9800 y=-0.2800 488 a=4.8800 77 x=-3.8200 y=-0.2600 501 a=5.9000	(قَبۡلِكَ)
Mx 	182.0124
Mx 	7.3800
Mx 	5.9800
Mx 	11.5800
Mx 	6.8000
Mx 	6.0400
Mx 	5.5600
Mx 	5.7000
Mx 	9.6600
T	79 x=-1.2200 45 a=7.3800 77 x=-1.4400 y=-0.2600 884 a=5.9800 79 x=0.5400 880 a=11.5800 679 a=6.8000 77 x=-5.3600 y=-0.2600 1260 118 x=0.2600 y=-0.2800 677 a=6.0400 462 a=5.5600 79 x=-2.2400 1012 a=5.7000 77 x=0.4800 y=-0.2600 71 a=8.0000 x=1.6600	(وَبِٱلۡءَاخِرَةِ)
Mx 	151.5089
Mx 	10.4400
Mx 	9.0400
T	118 x=0.9800 y=-0.2800 527 a=10.4400 78 x=-2.5800 y=-0.3800 519 a=9.0400	(هُمۡ)
Mx 	100.6654
Mx 	11.4400
Mx 	6.9000
Mx 	4.8800
Mx 	5.9000
Mx 	6.9000
Mx 	3.8000
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-3.8400 y=-0.3800 490 a=4.8800 79 x=-2.3000 501 a=5.9000 555 a=6.9000 78 x=-4.3200 y=-0.3800 484 a=3.8000	(یُوقِنُونَ)
Mx 	94.8254
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1266 a=5.8400 x=-15.7200	(٤)
Mx 	69.2454
T	114 w=25.5800	(۝)
Mx 	14.8819
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	16.4400
Mx 	4.8800
Mx 	5.8400
Mx 	3.5000
Mx 	8.0000
Mx 	4.6800
T	77 x=1.7400 y=-0.2600 521 a=16.4400 373 x=2.6400 y=-2.5000 84 x=2.6400 y=3.0200 493 a=4.8800 121 x=1.9600 y=-2.5200 1258 a=5.8400 77 x=-4.6000 y=-0.2600 525 a=3.5000 116 x=0.5600 71 a=8.0000 371 x=1.8600 y=2.7400 39 a=4.6800	(أُو۟لَـٰۤئِكَ)
New page
Mx 	261.6759
My 	56.0364
Mx 	9.3800
Mx 	3.9000
Mx 	7.8000
T	101 x=2.0000 y=-16.0200 661 a=9.3800 77 x=-5.3800 y=-0.2600 660 a=3.9000 77 x=-3.1400 y=-0.2600 969 a=7.8000	(عَلَىٰ)
Mx 	222.8746
Mx 	15.2800
Mx 	9.4200
Mx 	9.0400
T	72 a=15.2800 141 x=-0.3400 y=-0.4000 497 a=9.4200 78 x=-2.5800 y=-0.3800 519 a=9.0400	(هُدࣰى)
Mx 	197.7332
Mx 	12.3000
Mx 	7.7800
T	530 a=12.3000 79 x=-1.3200 80 x=-0.2600 y=-0.4600 528 a=7.7800	(مِّن)
Mx 	161.9719
Mx 	10.4400
Mx 	2.9400
Mx 	7.6800
Mx 	9.6400
T	107 118 x=0.9800 y=-0.2800 527 a=10.4400 79 x=-3.2600 1175 a=2.9400 79 x=-0.2200 80 x=1.1400 y=-0.4600 1156 a=7.6800 369 x=5.4400 y=0.2200 80 x=1.4000 y=-0.4600 53 a=7.9800 x=1.6600	(رَّبِّهِمۡۖ)
Mx 	105.5706
Mx 	16.4400
Mx 	4.8800
Mx 	5.8400
Mx 	3.5000
Mx 	8.0000
Mx 	4.6800
Mx 	8.0000
T	77 x=1.7400 y=-0.2600 521 a=16.4400 373 x=2.6400 y=-2.5000 84 x=2.6400 y=3.0200 493 a=4.8800 121 x=1.9600 y=-2.5200 1258 a=5.8400 77 x=-4.6000 y=-0.2600 525 a=3.5000 116 x=0.5600 71 a=8.0000 371 x=1.8600 y=2.7400 39 a=4.6800 77 x=-1.1800 y=-0.2600 71 a=8.0000	(وَأُو۟لَـٰۤئِكَ)
Mx 	81.0292
Mx 	10.4400
Mx 	9.0400
T	78 x=-0.8800 y=-0.3800 527 a=10.4400 78 x=-2.5800 y=-0.3800 519 a=9.0400	(هُمُ)
Mx 	16.8279
Mx 	11.4400
Mx 	6.9000
Mx 	12.2000
Mx 	3.9400
Mx 	7.5600
Mx 	9.2600
Mx 	3.5000
Mx 	4.3400
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=0.8400 y=-0.3800 515 a=12.2000 79 x=-3.3600 526 a=3.9400 118 x=0.7600 y=-0.2800 506 a=7.5600 78 x=-1.6000 y=-0.3800 529 a=9.2600 118 x=-2.8600 y=-0.2800 525 a=3.5000 102 a=4.3400	(ٱلۡمُفۡلِحُونَ)
Mx 	10.9879
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1267 a=5.8400 x=-15.7200	(٥)
Mx 	-14.5921
T	114 w=25.5800	(۝)
Mx 	266.9759
My 	131.0364
Set font 	Amiri Quran;20;400;;normal;;RTL
Mx 	11.4400
Mx 	4.3400
T	369 x=5.2600 y=0.2200 80 x=1.2200 y=-0.4600 69 a=11.4400 373 x=3.2200 y=-1.4200 41 a=4.3400	(إِنَّ)
Mx 	227.8269
Mx 	11.2400
Mx 	2.9000
Mx 	9.7600
Mx 	2.8200
Mx 	4.3400
T	77 x=1.0600 y=-0.2600 726 a=11.2400 725 a=2.9000 79 x=-0.9000 1071 a=9.7600 369 x=-0.8600 y=0.2200 80 x=-4.9000 y=-0.4600 1067 a=2.8200 102 a=4.3400	(ٱلَّذِینَ)
Mx 	179.7979
Mx 	4.3400
Mx 	8.0000
Mx 	6.8600
Mx 	7.5600
Mx 	13.1800
T	116 x=-1.6800 y=-0.8800 43 a=4.3400 71 a=8.0000 78 x=-3.5400 y=-0.3800 535 a=6.8600 77 x=-0.9800 y=-0.2600 506 a=7.5600 77 x=0.9600 y=-0.2600 522 a=13.1800	(كَفَرُوا۟)
Mx 	138.5689
Mx 	7.6200
Mx 	7.2600
Mx 	6.9000
Mx 	11.3600
T	75 x=-2.0400 y=-9.6400 37 a=7.6200 121 x=3.4200 y=-1.0600 43 a=4.3400 x=1.4600 77 x=-2.4600 y=-0.2600 555 a=6.9000 77 x=0.9600 y=-0.2600 544 a=11.3600	(سَوَاۤءٌ)
Mx 	99.1399
Mx 	10.4400
Mx 	2.9400
Mx 	6.2200
Mx 	3.9400
Mx 	7.8000
T	118 x=0.9800 y=-0.2800 527 a=10.4400 79 x=-3.2600 1175 a=2.9400 118 x=0.1600 y=-0.2800 1174 a=6.2200 77 x=-4.1200 y=-0.2600 526 a=3.9400 77 x=-3.1400 y=-0.2600 969 a=7.8000	(عَلَیۡهِمۡ)
Mx 	36.6909
Mx 	10.4400
Mx 	2.9400
Mx 	7.6800
Mx 	7.9800
Mx 	9.6200
Mx 	3.4000
Mx 	4.6800
Mx 	7.6200
T	118 x=0.9800 y=-0.2800 527 a=10.4400 78 x=-3.4200 y=-0.3800 1175 a=2.9400 77 x=-0.0800 y=-0.2600 1159 a=7.6800 118 x=0.2600 y=-0.2800 53 a=7.9800 77 x=-1.1600 y=-0.2600 617 a=9.6200 611 a=3.4000 1286 x=2.2600 y=-0.3400 39 a=4.6800 77 x=-2.4600 y=-9.4400 37 a=7.6200	(ءَأَنذَرۡتَهُمۡ)
Mx 	14.8819
Mx 	9.0400
Mx 	4.6800
T	118 x=0.2600 y=-0.2800 68 a=9.0400 1286 x=2.2600 y=-0.3400 39 a=4.6800	(أَمۡ)
Mx 	268.3359
My 	206.0364
Mx 	9.6600
Mx 	4.7600
T	118 x=0.2600 y=-0.2800 1294 a=9.6600 77 x=-3.4200 y=0.4200 1121 a=4.7600	(لَمۡ)
Mx 	214.6788
Mx 	10.4400
Mx 	9.0400
Mx 	7.9800
Mx 	9.4200
Mx 	3.3600
Mx 	6.3800
T	118 x=0.9800 y=-0.2800 527 a=10.4400 78 x=-2.5800 y=-0.3800 519 a=9.0400 118 x=0.2600 y=-0.2800 53 a=7.9800 79 x=-0.3400 496 a=9.4200 688 a=3.3600 78 x=-3.0600 y=-0.3800 821 a=6.3800	(تُنذِرۡهُمۡ)
Mx 	194.8016
Mx 	6.8000
Mx 	6.0400
T	679 a=6.8000 77 x=-1.4800 y=-0.2600 677 a=6.0400	(لَا)
Mx 	146.0645
Mx 	11.4400
Mx 	6.9000
Mx 	4.8800
Mx 	7.7800
Mx 	6.9000
Mx 	3.8000
T	77 y=-0.2600 69 a=11.4400 555 a=6.9000 78 x=-3.8400 y=-0.3800 490 a=4.8800 79 x=-1.3200 528 a=7.7800 118 x=-0.7200 y=-0.2800 554 a=6.9000 78 x=-4.3200 y=-0.3800 484 a=3.8000	(یُؤۡمِنُونَ)
Mx 	140.2245
T	22 w=5.8400	( )
Set font 	Amiri Quran;20;400;;normal;;LTR
T	1268 a=5.8400 x=-15.7200	(٦)
Mx 	114.6445
T	114 w=25.5800	(۝)
End page
Finish
